shell/platform/darwin/macos/framework/Source/FlutterEngineTest.mm

```diff
@@ -535,7 +535,56 @@ @implementation MockableFlutterEngine
   // TODO(gw280): add support for screenshot tests in this test harness
 
   [engine shutDownEngine];
-}  // namespace flutter::testing
+}
+
+TEST_F(FlutterEngineTest, CompositorIgnoresUnknownView) {
```

Comment:
Shouldn't this crash due to the `FML_DCHECK`?

Reply:
Right! I've removed the `FML_DCHECK`. It was a second thought and I didn't run the test after adding it.